ImprovedAnvil/lib/core/2da.tph

```diff
@@ -203,6 +210,32 @@ COPY_EXISTING ~luabbr.2da~  ~override/luabbr.2da~
   REPLACE_TEXTUALLY ~BEAST_FRIEND.+Dr0~ ~BEAST_FRIEND              Dr3~
   PRETTY_PRINT_2DA
 
+// assign new ludr file to the Priest of Talos kit
```

Comment:
these five changes could be combined into a single one

Reply:
Combined them into one change